src/server/paths.js

```diff
@@ -90,11 +89,6 @@ export const formatEntryPaths = (entry) => {
 export const formatSpecPaths = (entry) => {
   const paths = formatEntryPaths(entry);
 
-  // Parse the specs.
-  bdd.register();
-  paths.forEach(path => require(path));
-  bdd.unregister();
-
```

Comment:
FYI - this was originally done so that we would have the specs running on the server for lookups when doing `client-to-server` calls.

This was a feature I had in an old UIHarness that was really helpful, but hadn't been implemented in the new one (yet).

We can figure out another way to do this later when/if we need to.  For now, getting TS in is more important!